packages/errors/src/__tests__/instruction-error-test.ts

```diff
@@ -95,11 +96,10 @@ describe('getSolanaErrorFromInstructionError', () => {
             }),
         );
     });
-    it('produces the correct `SolanaError` for a `BorshIoError` error', () => {
+    it('produces the correct `SolanaError` for a `BorshIoError` error from a string', () => {
         const error = getSolanaErrorFromInstructionError(123, { BorshIoError: 'abc' });
```

Comment:
This change means that we need to make two tests, right: one that handles the error being `{ BorshIoError: 'abc' }` and a second test that handles it being `"BorshIoError"`?

Reply:
I added it on line 54 as part of the generic test -- shouldn't that do what you're suggesting?